mail/2.1/_index.md

```diff
@@ -18,16 +18,16 @@ The JDK version required will be aligned with Jakarta EE 10.
 The Specification Committee Ballot concluded successfully on 2021-xx-xx with the following results.
 | Representative                     | Representative for: |  Vote  |
 |------------------------------------|---------------------|--------|
-| Kenji Kazumura                     | Fujitsu             |      |
-| Dan Bandera, Kevin Sutter          | IBM                 |      |
-| Ed Bratt, Dmitry Kornilov          | Oracle              |      |
-| Andrew Pielage, Matt Gill          | Payara              |      |
-| Scott Stark, Mark Little           | Red Hat             |      |
-| David Blevins, Jean-Louis Monteiro | Tomitribe           |      |
-| Ivar Grimstad                      | EE4J PMC            |      |
-| Marcelo Ancelmo, Martijn Verburg   | Participant Members |      |
-| Werner Keil                        | Committer Members   |      |
-| Scott (Congquan) Wang              | Enterprise Members  |      |
-|                                    | **Total**           | **** |
+| Kenji Kazumura                     | Fujitsu             | +1     |
+| Dan Bandera, Kevin Sutter          | IBM                 | +1     |
+| Ed Bratt, Dmitry Kornilov          | Oracle              | +1     |
+| Andrew Pielage, Matt Gill          | Payara              | +1     |
+| Scott Stark, Mark Little           | Red Hat             | +1     |
+| David Blevins, Jean-Louis Monteiro | Tomitribe           | +1     |
+| Ivar Grimstad                      | EE4J PMC            | +1     |
+| Marcelo Ancelmo, Martijn Verburg   | Participant Members | +1     |
+| Werner Keil                        | Committer Members   | +1     |
+| Dr. Jun Qian                       | Enterprise Members  | +1     |
+|                                    | **Total**           | 10     |
 The ballot was run in the [jakarta.ee-spec mailing list]()
```

Comment:
@Pandrex247 Need to fill in the reference for the Spec Mailing List url.  Also, at the top of this subsection (line 18), you need to update this line with the date of the successful ballot.  Thanks!